### DISCUSSION TITLE
database

### DISCUSSION THREAD

**Discussion by ahmed26399** [atrocore/atrocore#561] [Q&A] [answered]
Hello,
I am trying to use AtroCore. I want to know is there any obligation to use  only MySQL. I want to know  if I can use other database like PostgreSQL or Oracle.
Thanks in advance

**Comment by atrolex** [comment, marked as answer]
Currently only MySQL is possible. We plan to onboard PostgreSQL soon, so the user can decide during installation which DB engine should be used.
